charts/turing/templates/_config.tpl

```diff
@@ -0,0 +1,38 @@
+{{- define "turing.renderedConfig" -}}
+{{- $ := index . 0 }}
+{{- $rendered := index . 2}}
+{{- if $rendered }}
+{{- $tag := $rendered.releasedVersion}}
+{{- $ensemblerTag := $rendered.ensemblerTag }}
+{{- $imagePrefix := "ghcr.io/caraml-dev/turing/pyfunc-ensembler-job-py" -}}
+{{- $imagePrefix2 := "ghcr.io/caraml-dev/turing/pyfunc-ensembler-service-py" -}}
```

Comment:
nit: can we have `imagePrefix` renamed as something like `ensemblerJobPrefix` or maybe just `jobPrefix`? Same thing for `imagePrefix2`, so it becomes `servicePrefix`?

Reply:
Fixed